doc/python/hardware.rst

```diff
@@ -193,6 +193,7 @@ The Python wrapper provides an API for:
         using :py:func:`~arbor.env.default_allocation()`.
 
     .. method:: context(threads=threads, gpu_id=gpu_id, mpi=mpi_comm, inter=mpi_comm)
+        :noindex:
```

Comment:
Why?

Reply:
The method above has the same name (overload) and sphinx issues a warning that the method is already indexed.